src/decorator/decoratorConfig.ts

```diff
@@ -1,38 +1,51 @@
-import { DecoratorProps, EnhetDisplay, FnrDisplay } from './decoratorProps';
-
-const RESET_VALUE = '\u0000';
+import {
+  DecoratorProps,
+  Enhet,
+  Environment,
+  UrlFormat,
+} from './decoratorProps';
+import { erAnsattDev, erDev, erLokal, erProd } from '@/utils/miljoUtil';
```

Comment:
Må vel gjøres noen tilpasninger i `miljoUtil` for at dette skal fungere

Reply:
Ja, fikset nå.